cocotb/share/makefiles/Makefile.sim

```diff
@@ -58,6 +58,7 @@ WAVES                     Enable wave traces dump for Riviera-PRO and Questa
 VERILOG_SOURCES           A list of the Verilog source files to include
 VHDL_SOURCES              A list of the VHDL source files to include
 VHDL_SOURCES_<lib>        VHDL source files to include in *lib* (GHDL/ModelSim/Questa/Xcelium only)
+VHDL_LIB_ORDER            Order in which VHDL libraries should be compiled (ModelSim/Questa/Xcelium only)
```

Comment:
Could GHDL also be supported?

```suggestion
VHDL_LIB_ORDER            Compilation order of VHDL libraries (ModelSim/Questa/Xcelium only)
```

Reply:
It could be, but doesn't need to be. GHDL orders sources automatically (like `-smartorder`, but apparently it actually works).

Reply:
Good to hear, then maybe
```suggestion
VHDL_LIB_ORDER            Compilation order of VHDL libraries (needed for ModelSim/Questa/Xcelium)
```